cypress/support/e2e.js

```diff
@@ -0,0 +1,22 @@
+
+// ***********************************************************
+// This example support/e2e.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import 'cypress-plugin-api'
+import 'cypress-axe'
```

Comment:
```suggestion
import 'cypress-axe'

```

É uma boa prática e convenção deixar uma linha em branco entre _libs_ externas e módulos internos.

A propósito, caso você delete o arquivo `cypress/commands/commands.js`, conforme sugerido dois comentários atrás, lembre de deletar a linha abaixo completamente.